internal/controller/pkg/manager/reconciler.go

```diff
@@ -183,7 +183,7 @@ func SetupProvider(mgr ctrl.Manager, o controller.Options) error {
 		WithNewPackageFn(np),
 		WithNewPackageRevisionFn(nr),
 		WithNewPackageRevisionListFn(nrl),
-		WithRevisioner(NewPackageRevisioner(f, WithDefaultRegistry(o.DefaultRegistry))),
```

Comment:
does strict validation option need to be provided in the statements in this file too?

Reply:
package revisioner, which afaict is the only component parsing the image, is set to enforce strict validation.